content/en/docs/languages/cpp/quickstart.md

```diff
@@ -56,71 +62,70 @@ Check the version of `cmake`:
 
 ```sh
 $ cmake --version
+cmake version {{< param cmake-version >}}
 ```
 
-Under Linux, the version of the system-wide `cmake` can be too low. You can
-install a more recent version into your local installation directory as follows:
+Under Linux, the version of the system-wide `cmake` can often be too old. You
+can install a more recent version into your local installation directory as
+follows:
 
 ```sh
 $ wget -q -O cmake-linux.sh https://github.com/Kitware/CMake/releases/download/v{{< param cmake-version >}}/cmake-{{< param cmake-version >}}-Linux-x86_64.sh
 $ sh cmake-linux.sh -- --skip-license --prefix=$MY_INSTALL_DIR
 $ rm cmake-linux.sh
 ```
 
+#### Install other required tools
 
-#### gRPC and Protocol Buffers
-
-While not mandatory, gRPC applications usually leverage [Protocol Buffers][pb]
-for service definitions and data serialization, and the example code uses
-[proto3][].
+Install the basic tools required to build gRPC:
 
-The following instructions will locally install gRPC and Protocol Buffers.
+- Linux
 
- 1. Install the basic tools required to build gRPC:
+  ```sh
+  $ sudo apt install -y build-essential autoconf libtool pkg-config
+  ```
 
-    - Linux
+- macOS:
 
-      ```sh
-      $ sudo apt install -y build-essential autoconf libtool pkg-config
-      ```
+  ```sh
+  $ brew install autoconf automake libtool pkg-config
+  ```
 
-    - macOS:
+#### Clone the `grpc` repo
 
-      ```sh
-      $ brew install autoconf automake libtool pkg-config
-      ```
+Clone the `grpc` repo and its submodules:
 
- 2. Clone the `grpc` repo and its submodules:
+```sh
+$ git clone --recurse-submodules -b {{< param grpc_vers.core >}} https://github.com/grpc/grpc
+```
+#### Build and install gRPC and Protocol Buffers
 
-    ```sh
-    $ git clone --recurse-submodules -b {{< param grpc_vers.core >}} https://github.com/grpc/grpc
-    $ cd grpc
-    ```
+While not mandatory, gRPC applications usually leverage [Protocol Buffers][pb]
+for service definitions and data serialization, and the example code uses
+[proto3][].
 
- 3. Build and locally install gRPC and all requisite tools:
+The following commands will build and locally install gRPC and Protocol Buffers.
```

Comment:
nit: add a note that we always recommend installing under a CMAKE_INSTALL_PREFIX when using "make install" since there is not easy way to undo "make install" and it's easy to pollute your system with a global non-prefixed install?

Alternatively we can say that we strongly discourage using "make install" globally, without using a prefix.

Reply:
Sounds good. I'll commit this as is and add a note in a followup PR -- since we didn't have such a note before, it's just an addition that is independent of the edits done in this PR.